swarmzero/agent.py

```diff
@@ -443,6 +444,8 @@ def _assign_agent(self, tools, tool_retriever):
                 agent_class = OpenAIMultiModalLLM
             elif "gpt" in model:
                 agent_class = OpenAILLM
+            elif "azure/" in model:
```

Comment:
great first PR @talrejanikhil!

why the trailing `/` after `azure`?

Reply:
This is to distinguish between regular OpenAI from Azure. In Azure the deployment names could be same as model names and then we will not be able to tell them apart here in this line. This is also a standard way to setup Azure OpenAI in other frameworks :)